packages/affiliates/test/mocha/affiliates.js

```diff
@@ -97,12 +119,17 @@ describe("DeployerRewards", function() {
       endTime: endingTimestamp,
       empWhitelist: empContracts,
       empCreatorAddress: empCreator,
-      tokensToPrice: syntheticTokens,
-      tokenDecimals: syntheticTokenDecimals
+      collateralTokens: collateralTokens,
+      collateralTokenDecimals: collateralTokenDecimals,
+      syntheticTokenDecimals: syntheticTokenDecimals
     });
-    // WIP
 
-    assert.equal(Object.keys(result).length, 1); // there should only be 1 deployer
-    assert.equal(Number(Object.values(result)[0]), Number(devRewardsToDistribute)); // the total rewards distributed should equal the number specified
+    assert.equal(Object.keys(result).length, 2); // There should be 2 deplorers for the 3 EMPs.
+    assert.equal(
```

Comment:
there are now two contract deployers for the 3 emps. check the sum adds up as expected.

Reply:
awesome yea im glad you did this